frontend/src/components/Breadcrumb.tsx

```diff
@@ -0,0 +1,45 @@
+import { FC } from 'react'
+import Link from 'next/link'
+
+type BreadcrumbItemType = {
+    text: string;
+    link: string;
+  };
+
+export interface BreadcrumbProps {
+    id?: string;
+    items?: BreadcrumbItemType[];
```

Comment:
I think the breadcrumbs should be required to always have at least one item.

Reply:
Breadcrumb props don't exist anymore as the breadcrumbs are auto-generated.